package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-pay-statement-constructor",
-  "version": "1.1.0",
+  "version": "1.1.1",
```

Comment:
This involves database change, may consider version 1.2.0

Reply:
True, does change the behaviour of received requests so guess that's a new feature and therefore minor change.  Updated.